tests/utils/test_rest_utils.py

```diff
@@ -202,6 +202,12 @@ def test_http_request_wrapper(request):
     request.assert_called_with(
         url="http://my-host/my/endpoint", verify=False, headers=_DEFAULT_HEADERS,
     )
+    response.text = "non json"
+    request.return_value = response
+    http_request_safe(host_only, "/my/endpoint")
+    request.assert_called_with(
+        url="http://my-host/my/endpoint", verify=False, headers=_DEFAULT_HEADERS,
```

Comment:
@smurching Added a test here to verify that `http_request_safe` doesn't throw for a non-mlflow-rest-api endpoint that returns a non-json response.